ideasbox/static/ideasbox/main.css

```diff
@@ -313,7 +313,33 @@ html[dir='rtl'] .boxid {
     float: left;
     right: -31px;
 }
-
+div.i18n_switch select {
```

Comment:
Better not to use `elem.class child` but `.class child` instead, as explained in [Don’t qualify class rules with tag names](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Writing_efficient_CSS#Don.27t_qualify_class-categorized_rules_with_tag_names).

So I would write: `.i18n_switch select` here.